src/Components/FilterBursaries.jsx

```diff
@@ -0,0 +1,37 @@
+//NPM packages
+
+//Project files
+import BursaryItem from "./BursaryItem";
+import bursaries from "../Data/bursaries.json";
+import { useFilter } from "state/FilterContext";
+
+export default function FilterBursaries() {
```

Comment:
A component shouldn't have its functionality (filter) in the naming. If you add or remove functionality you would have to rename the component which becomes a chore
```suggestion
export default function Bursaries() {
```